offlinedocs/scripts/copyImages.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cp ../docs/images public/images --recursive
```

Comment:
Should `rm` first so any deleted images get deleted from here as well

Reply:
Do you mean something like `rm -rf public/images && cp ../docs/images public/images --recursive`?

Reply:
Yeah, but it can go on separate lines instead of using `&&` since it's a shell file